Tasks/AzureKeyVaultV1/package.json

```diff
@@ -6,10 +6,10 @@
     "@types/node": "^10.17.0",
     "@types/q": "^1.5.0",
     "@types/uuid": "^8.3.0",
-    "azure-devops-node-api": "11.2.0",
-    "azure-pipelines-task-lib": "^4.0.1-preview",
-    "azure-pipelines-tasks-azure-arm-rest-v2": "^3.213.0",
-    "moment": "^2.29.4"
+    "azure-pipelines-task-lib": "2.8.0",
+    "azure-pipelines-tasks-azure-arm-rest-v2": "1.0.4",
```

Comment:
This looks a bit suspicious but I didn't dig deeper. 
I would expect 2.198.2 here
https://github.com/microsoft/azure-pipelines-tasks/commit/4718ac9183db21ba1e7526dea31dfb4ac9e67679

Reply:
There was 1.0.4 originally. I mean in the previous version before #17225.